src/logic/section.model.ts

```diff
@@ -6,7 +6,22 @@ export enum ChildrenSectionKey {
 }
 
 export enum ExtraWorkersSectionKey {
-  ExtraWorkersCount = "liczba dodatkowych pracowników",
+  ExtraWorkersCount = "pracownicy dzienni",
+}
+
+export class FoundationSectionKey {
```

Comment:
Klasa, ponieważ chcieliśmy wykorzystać klucze z innego enuma, a VS Code wyrzucał ostrzeżenie o tym, że nie da się tego zrobić.  Ale okazuję się, że da:
![image](https://user-images.githubusercontent.com/21079319/100127135-1734e980-2e7f-11eb-94b0-2af94625f68b.png)

Więc można było by to zbadać. Można też zostawić tak jak jest, bo osobiście mi to nie za bardzo przeszkadza.